src/clj/game/core/change_vals.clj

```diff
@@ -24,13 +24,13 @@
   (gain state side key {:mod delta})
   (change-msg state side key (base-mod-size state side key) delta))
 
-(defn- change-mu
-  "Send a system message indicating how mu was changed"
-  [state side delta]
-  (free-mu state delta)
-  (system-msg state side
-              (str "sets unused MU to " (available-mu state)
-                   " (" (if (pos? delta) (str "+" delta) delta) ")")))
+; (defn- change-mu
+;   "Send a system message indicating how mu was changed"
+;   [state side delta]
+;   (free-mu state delta)
+;   (system-msg state side
+;               (str "sets unused MU to " (available-mu state)
+;                    " (" (if (pos? delta) (str "+" delta) delta) ")")))
```

Comment:
Any reason to leave this in and commented out rather than just deleting?

Reply:
Whoops, I forgot to rewrite it. Lol thanks for catching this.